adsk-auth.js

```diff
@@ -41,19 +41,19 @@ export class AdskAuth extends events.EventEmitter {
 		if (this.timeoutId)
 			clearTimeout(this.timeoutId);
 
-		let dataString = "client_id=" + this.key
-			+ "&client_secret=" + this.secret
-			+ "&grant_type=client_credentials"
+		let dataString = "grant_type=client_credentials" 
 			+ "&scope=" + scope;
 
 		let headers = {
-			"Content-Type": "application/x-www-form-urlencoded"
+			"Content-Type": "application/x-www-form-urlencoded",
+			"Accept": "application/json",
+			"Authorization": `Basic ${btoa(`${this.key}:${this.secret}`)}`,
```

Comment:
Guess we could start specifying a minimum node.js version (16 I guess for btoa) in package.json...